core/shared/src/test/scala/fs2/CompressionSuite.scala

```diff
@@ -82,6 +82,36 @@ abstract class CompressionSuite(implicit compression: Compression[IO]) extends F
     )
   )
 
+  test("inflate please wrap") {
```

Comment:
Added these tests because actually the existing Node.js inflate/deflate implementation was broken 😅